dev/udf_proxy.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <sqlite3.h>
-#include <cassert>  //  assert macro
+#include <assert.h>  //  assert macro
```

Comment:
why? I was sure that C++ style for C includes is preferred in C++

Reply:
Not with macros. If one is importing the Standard Library as a module you need to include the C header to get macros.